<launch>

   <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video1" />   
      <param name="image_width" value="1280" />
      <param name="image_height" value="720" />
      <param name="pixel_format" value="mjpeg" />
      <param name="camera_frame_id" value="/camera_link" />
      <param name="framerate" value="30" />
      <param name="contrast" value="32" />
      <param name="brightness" value="32" />
      <param name="saturation" value="32" />
      <param name="autofocus" value="false" />
      <param name="focus" value="51" />
      <param name="camera_info_url" value="package://line_detection/config/head_camera.yaml" /> 
      <param name="camera_name" value="head_camera" />
      <param name="io_method" value="mmap"/>
    </node>
<!--
  <node name="bag_player" pkg="rosbag" type="play" args="-r 0.5 --clock $(find zbar_decoder)/slow_line_follow_with_qr.bag"/> 
  <rosparam param="use_sim_time" > true </rosparam>
-->
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  
  <!-- <include file="$(find zbar_decoder)/launch/zbar_decoder_node.launch" />  -->
</launch>
